package.json

```diff
@@ -7,20 +7,21 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "babel-eslint": "8.2.3",
-    "eslint": "4.19.1",
-    "eslint-config-airbnb": "16.1.0",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-import-resolver-typescript": "^1.0.2",
-    "eslint-import-resolver-webpack": "0.10.0",
-    "eslint-plugin-flowtype": "2.49.3",
-    "eslint-plugin-flowtype-errors": "3.6.0",
-    "eslint-plugin-import": "2.12.0",
-    "eslint-plugin-jsx-a11y": "5.1.1",
-    "eslint-plugin-no-only-tests": "2.0.0",
-    "eslint-plugin-react": "7.12.4",
-    "eslint-plugin-typescript": "^0.12.0",
-    "typescript-eslint-parser": "^18.0.0"
+    "babel-eslint": "10.0.2",
+    "eslint": "6.1.0",
+    "eslint-config-airbnb": "17.1.1",
+    "eslint-config-prettier": "^6.0.0",
+    "eslint-import-resolver-typescript": "^1.1.1",
+    "eslint-import-resolver-webpack": "0.11.1",
+    "eslint-plugin-flowtype": "3.12.2",
+    "eslint-plugin-flowtype-errors": "4.1.0",
+    "eslint-plugin-import": "2.18.2",
+    "eslint-plugin-jsx-a11y": "6.2.3",
+    "eslint-plugin-no-only-tests": "2.3.1",
+    "eslint-plugin-react": "7.14.3",
+    "eslint-plugin-react-hooks": "^1.6.1",
+    "eslint-plugin-typescript": "^0.14.0",
+    "typescript-eslint-parser": "^22.0.0"
```

Comment:
i think we said, that we dont want to use the `^`, especially not for those, that start with `0.`

@TkDodo ?

Reply:
yes, please no `^`

Reply:
0317a4f